.travis.yml

```diff
@@ -1,9 +1,18 @@
 language: php
+
 php:
   - 5.6
   - 5.5
-before_script:
+
+cache:
+  directories:
+    - $HOME/.composer/cache
```

Comment:
Caching dependencies as installed with `composer` between builds.

For reference, see https://docs.travis-ci.com/user/caching/.